meta-ostro/classes/ostro-image-extensions.bbclass

```diff
@@ -0,0 +1,15 @@
+# Class that provides additional tools and methods for image conversions.
```

Comment:
I would drop "ostro" from the name of the class. There isn't really anything specific to ostro.